app/controllers/emails_controller.rb

```diff
@@ -54,15 +54,22 @@ def save_template
       flash.now[:alert] = t('.saving_failed')
     end
     @event = Event.find(params[:event_id])
-    @templates = EmailTemplate.with_status(get_status)
+    @templates = EmailTemplate.with_status(get_email_template_status)
 
     render :email
   end
 
-  def get_status
+  def get_email_template_status
     params[:status] ? params[:status].to_sym : :default
   end
 
+  def get_corresponding_application_letter_status
```

Comment:
Currently maybe it looks a little bit overkill but a unit test should be provided, shouldn't it?